src/js/components/CheckBox/stories/Reverse.js

```diff
@@ -7,14 +7,19 @@ const SimpleCheckBox = ({ checked: checkedProp, ...rest }) => {
   const onChange = (event) => setChecked(event.target.checked);
 
   return (
-    <Box align="center" pad="large">
+    <Box gap="small" align="center" pad="large">
       <CheckBox {...rest} checked={checked} onChange={onChange} />
     </Box>
   );
 };
 
-export const Reverse = () => <SimpleCheckBox label="Choice" reverse />;
+export const Simple = () => (
+  <>
+    <SimpleCheckBox label="Reversed Checkbox" reverse />
+    <SimpleCheckBox label="Disabled Checkbox" checked disabled />
+  </>
+);
 
 export default {
-  title: 'Input/CheckBox/Reverse',
+  title: 'Input/CheckBox/Simple',
```

Comment:
We should change the file name to be 'Simple' instead of 'Reverse'

Reply:
oh yes forgot that bit